### DISCUSSION TITLE
A "what are the core team up to" page

### DISCUSSION THREAD

**Discussion by aavmurphy** [twbs/bootstrap#36840] [Ideas] [answered]
new releases are are a (nice) surprise
it would be nice to know what you're working on..., what's coming next, maybe a monthly update
for example,  will there be a 5.2 bug fix release, e.g, for #36813 (a tooltip regression), or are you all on the beach this month
andrew

**Comment by julien-deramond** [comment]
You can follow what we're working on by checking [our projects](https://github.com/twbs/bootstrap/projects) and the different issues and PRs. It answers your question about the 5.2 bug fix release ([v5.2.1 project](https://github.com/orgs/twbs/projects/8)).

That doesn't prevent us to go to the beach this summer :beach_umbrella: :slightly_smiling_face:

**Comment by aavmurphy** [comment]
Thanks for the bug-fix answer.
Have looked at the "project" pages before, but its hard to get a clear idea of the big picture from them.
A 1-page summary/roadmap would be nice :)

**Comment by mdo** [comment, marked as answer]
Here's a little rundown of what's coming:

- Yes, there will be a patch release for v5.2.1. Might try to do this remotely (for me) this week, otherwise it'll be next week.
- v5.3.0 is the next minor release, the focus of which is dark mode (see #35857)
- v5.4.0 may come and could see some new JS components added (@GeoSot has some demos going). If not, this could go into v6, but depends on the team's bandwidth.
- Then likely onto v6.0.0 with some dependency updates, browser support changes, theming improvements, layout improvements, Sass modules (most likely), and other modern/future frontend improvements.
